src/chainstate/stacks/boot/contract_tests.rs

```diff
@@ -341,6 +341,7 @@ fn recency_tests() {
 fn delegation_tests() {
     let mut sim = ClarityTestSim::new();
     let delegator = StacksPrivateKey::new();
+    const REWARD_CYCLE_LENGTH: u128 = 1050;
```

Comment:
Want to calculate this with `POX_REWARD_CYCLE_LENGTH / 2` instead of using 1050?

Reply:
In this case, I don't think that it would improve readability